e2e/support/helpers/component-embedding-sdk-helpers.tsx

```diff
@@ -0,0 +1,77 @@
+import type { SDKConfig } from "@metabase/embedding-sdk-react";
+import { MetabaseProvider } from "@metabase/embedding-sdk-react";
+import * as jose from "jose";
+import type { JSX } from "react";
+
+import { USERS } from "e2e/support/cypress_data";
+import { signInAsAdminAndEnableEmbeddingSdkForE2e } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+
+export const METABASE_INSTANCE_URL = "http://localhost:4000";
+
+export const JWT_PROVIDER_URL = "http://jtw-provider/sso";
```

Comment:
```suggestion
export const JWT_PROVIDER_URL = "http://jwt-provider/sso";
```

Reply:
Since we renamed the public field, to avoid confusion maybe we should use `auth-provider` here too?

Reply:
💯, updated the above snippet